packages/react-instantsearch-core/src/connectors/__tests__/connectSortBy.js

```diff
@@ -8,15 +8,12 @@ let params;
 
 describe('connectSortBy', () => {
   describe('single index', () => {
-    const context = { context: { ais: { mainTargetedIndex: 'index' } } };
-    const getProvidedProps = connect.getProvidedProps.bind(context);
-    const refine = connect.refine.bind(context);
-    const getSP = connect.getSearchParameters.bind(context);
-    const getMetadata = connect.getMetadata.bind(context);
-    const cleanUp = connect.cleanUp.bind(context);
+    const contextValue = { mainTargetedIndex: 'index' };
+
     it('provides the correct props to the component', () => {
-      props = getProvidedProps(
-        { items: [{ value: 'yep' }, { value: 'yop' }] },
+      props = connect.getProvidedProps.call(
```

Comment:
We don't have to use `.call` we don't rely on `this` inside the connector. We have to apply this change in the all file.

Reply:
I just realised I confused SortBy & ScrollTo